[WIP] Update router naming conventions (#871)
apollographql/docs: src/content/graphos/basics/graphs/updating.mdx

  end;
```

> - The **GraphOS Router** is powered by a particular version of the Apollo Router Core.

Reviewer: ```suggestion
- The **GraphOS Router** is based on a version of the Apollo Router Core.
```
